certified-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,965 @@
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "CSSAPI_EntityListSelect",
+              "value-path": "EntityNameForAPI",
+              "value-title": "EntityNameDisplay"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/APIAttributeDefinitionResponse"
+              }
+            }
+          },
+          "400": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/CSSAPICustomMessage"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "505": {
+            "description": "HttpVersionNotSupported",
+            "schema": {
+              "$ref": "#/definitions/CSSAPICustomMessage"
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/CSSAPI/v2/PowerAutomate/PollTriggers/{entityName}/NewRecords": {
+      "post": {
+        "tags": [
+          "Poll Triggers"
+        ],
+        "summary": "A scheduled trigger that query created (new) records on a given interval",
+        "description": "A scheduled trigger that query created (new) records on a given interval.",
+        "operationId": "CSSAPI_New_Records_Poll_Trigger",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-trigger": "batch",
+        "x-ms-trigger-metadata": {
+          "mode": "polling",
+          "kind": "query"
+        },
+        "parameters": [
+          {
+            "name": "entityName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Entity Name",
+            "x-ms-url-encoding": "single",
+            "description": "Table name to pull new record(s) from.",
+            "x-ms-dynamic-values": {
+              "operationId": "CSSAPI_EntityListUpdate",
+              "value-path": "EntityNameForAPI",
+              "value-title": "EntityNameDisplay"
+            }
+          },
+          {
+            "name": "since",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "date-time",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "query",
+            "in": "body",
+            "description": "Query Model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/APIQueryRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@nizarhandal please provide a more detailed description if you have `default` description for responses..